XCUITests/XCUITests-Bridging-Header.h

```diff
@@ -0,0 +1,18 @@
+//
+//  XCUITests-Bridging-Header.h
```

Comment:
@farhanpatel Ah, I added this header to enable sending the app to the background in XCUITests, like I did in https://github.com/mozilla-mobile/firefox-ios/pull/2566/files#diff-30cb5c0e8f9990cd480c6e4d3be1af8dR104 This enables to grab XCUIApplication object for a non-target apps, like springboard.  You're right about the header, I'll update that